<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>601</width>
    <height>540</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>601</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuRecent_Files">
     <property name="title">
      <string>Recent Files</string>
     </property>
     <addaction name="actiondummy"/>
    </widget>
    <addaction name="actionOpen_File_s"/>
    <addaction name="menuRecent_Files"/>
    <addaction name="actionClose_File"/>
    <addaction name="actionClose_All_Files"/>
    <addaction name="separator"/>
    <addaction name="actionExport_Chart"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionColumns"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_BenchmarkViewer"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionOpen_File_s">
   <property name="text">
    <string>Open File(s)</string>
   </property>
  </action>
  <action name="actionClose_File">
   <property name="text">
    <string>Close File</string>
   </property>
  </action>
  <action name="actionClose_All_Files">
   <property name="text">
    <string>Close All Files</string>
   </property>
  </action>
  <action name="actionExport_Chart">
   <property name="text">
    <string>Export Chart</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionColumns">
   <property name="text">
    <string>Columns</string>
   </property>
  </action>
  <action name="actionAbout_BenchmarkViewer">
   <property name="text">
    <string>About BenchmarkViewer</string>
   </property>
  </action>
  <action name="actiondummy">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>dummy</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
